schema/metrics/histogram_metrics.mapping

```diff
@@ -0,0 +1,159 @@
+{
+  "index_patterns": [
+    "sso_metrics-*-*"
+  ],
+  "dynamic_templates": [
+    {
+      "attributes_map": {
+        "mapping": {
+          "type":"keyword"
+        },
+        "path_match":"attributes.*"
+      }
+    },
+    {
+      "resources_map": {
+        "mapping": {
+          "type": "keyword"
+        },
+        "path_match": "resources.*"
+      }
+    }
+  ],
+  "mappings": {
+    "_meta": {
+      "version": "0.1.0-dev"
+    },
+    "properties": {
+      "temporality": {
+        "type": "text",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "bucketCounts": {
+        "type": "long"
+      },
+      "bucketCountsList": {
+        "type": "long"
+      },
+      "buckets": {
+        "properties": {
+          "count": {
+            "type": "long"
+          },
+          "max": {
+            "type": "float"
+          },
+          "min": {
+            "type": "float"
+          }
+        }
+      },
+      "count": {
+        "type": "long"
+      },
+      "description": {
+        "type": "text",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "explicitBounds": {
```

Comment:
Same comment here - not possible to represent the fact that this is a list?

Reply:
same as above https://opensearch.org/docs/2.0/opensearch/supported-field-types/index/#arrays